_rules/video-captions-correct-a3b9xz.md

```diff
@@ -0,0 +1,160 @@
+---
+id: a3b9xz
+name: Video element auditory content has correct captions
+rules_format: 1.1
+rule_type: atomic
+description: |
+  This rule checks that provided captions are correct for audio information in non-streaming `video` elements.
+accessibility_requirements:
+  wcag20:1.2.2: # Captions (Prerecorded) (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:1.2.4: # Captions (Live) (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:F8: # Failure of Success Criterion 1.2.2 due to captions omitting some dialogue or important sound effects
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:F75: # Failure of Success Criterion 1.2.2 by providing synchronized media without captions when the synchronized media presents more information than is presented on the page
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Audio output
+  - Visual output
+acknowledgments:
+  authors:
+    - Kathy Eng
+  funding:
+    - WAI-Tools
+  assets:
+    - Web Accessibility Perspective videos by W3C WAI.
+---
+
+## Applicability
+
+This rule applies to [captions](https://www.w3.org/TR/WCAG22/#dfn-captions) provided for a [non-streaming](#non-streaming-media-element) `video` element that is [visible][] where the video contains [audio](#audio-output).
+
+## Expectation
+
+For each test target, speech and non-speech audio information that is not conveyed visually in the video is available in the [captions](https://www.w3.org/TR/WCAG22/#dfn-captions).
+
+**Note:** Captions can be either embedded in the video file itself or can be made available through a separate track.
+
+## Background
+
+### Assumptions
```

Comment:
You're making the assumption that a video with captions doesn't isn't also a media alternative. I think a video could in theory have incorrect captions, but still pass 1.2.2 if it's a media alternative. That's pretty theoretical though so I think an assumption about that is fine.